src/server.c

```diff
@@ -201,6 +201,10 @@ struct redisCommand redisCommandTable[] = {
      "read-only fast @string",
      0,NULL,1,1,1,0,0,0},
 
+    {"getex",getexCommand,-3,
+     "write use-memory @string",
```

Comment:
the EXPIRE and EXPIREAT commands don't have `use-memory` and that's the only effect of this command that can cause slight memory utilization.
it can be considered a `fast` command since even when it deletes a key, it can only be a string type
```suggestion
     "write fast @string",
```